libpay-braintree/src/test/scala/com/wix/pay/braintree/it/BraintreeGatewayWithOAuthIT.scala

```diff
@@ -0,0 +1,46 @@
+package com.wix.pay.braintree.it
+
+import com.braintreegateway.Environment
+import com.wix.pay.PaymentGateway
+import com.wix.pay.braintree._
+import com.wix.pay.creditcard.{CreditCard, YearMonth}
+import com.wix.pay.model.CurrencyAmount
+import org.specs2.mutable.SpecWithJUnit
+
+
+class BraintreeGatewayWithOAuthIT extends SpecWithJUnit {
```

Comment:
@ohadraz - are you guys ok that the test will use Braintree Sandbox Env ? i think its valid, if its starting failing due to networking i can add driver / wiremock